app/views/smart_answers/shared/_previous_answers.html.erb

```diff
@@ -1,54 +1,62 @@
+      text: "Change your answers",
+      heading_level: 2,
+      font_size: "l",
+      margin_bottom: 6
+    } %>
+  <% else %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Your answers",
+      heading_level: 2,
+      font_size: "m",
+      margin_bottom: 4
+    } %>
+  <% end %>
 
-      <p class="govuk-body">
-      <%= link_to "Start again",
-        restart_flow_path(@presenter),
-        :class => "govuk-link",
-        :data => {
-          module: "gem-track-click",
-          "track-action": "Start again",
-          "track-category": "StartAgain",
-          "track-label": @presenter.current_node.title
-        } %>
-      </p>
-    </caption>
+  <%= tag.p class: "govuk-body" do %>
+    <%= link_to "Start again",
+      restart_flow_path(@presenter),
+      class: "govuk-link",
+      data: {
+        module: "gem-track-click",
+        "track-action": "Start again",
+        "track-category": "StartAgain",
+        "track-label": @presenter.current_node.title
+      } %>
+  <% end %>
 
-    <% unless local_assigns[:hide_previous_answers] %>
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header">Question</th>
-          <th scope="col" class="govuk-table__header">Answer</th>
-          <th scope="col" class="govuk-table__header">Change answer</th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <% @presenter.collapsed_questions.each_with_index do |question, number_questions_so_far| %>
-          <%= render(
-              'smart_answers/shared/previous_answer',
-              question: question,
-              accepted_response: @presenter.accepted_responses[number_questions_so_far],
-              question_link: @presenter.change_collapsed_question_link(number_questions_so_far + 1, question),
-            )
-          %>
-        <% end %>
-      </tbody>
-    <% end %>
-  </table>
+  <% unless local_assigns[:hide_previous_answers] %>
+    <% items = @presenter.collapsed_questions.each_with_index.map { |question, number_questions_so_far|
+      accepted_response = @presenter.accepted_responses[number_questions_so_far]
+
+      if question.multiple_responses?
+        value = render "govuk_publishing_components/components/list", {
+          items: question.response_labels(accepted_response)
+        }
+      else
+        value = question.response_label(accepted_response)
+      end
+
+      {
+        field: question.title,
+        value: value,
+        edit: {
+          href: @presenter.change_collapsed_question_link(number_questions_so_far + 1, question),
+          data_attributes: {
+            "module": "gem-track-click",
+            "track-category": "Smart Answer Change Link",
+            "track-action":  "Link clicked",
+            "track-label": "#{question.title} / #{question.response_label(accepted_response)}"
+          }
+        }
+      }
```

Comment:
Nit: It'd be great to be able to push this logic into the presenter. If the question's knew their own responses, we wouldn't have to worry about matching up question presenters with their response, which would simplify things. 

But that's probably apart of a larger refactor.

Reply:
Actually, looks like this might be encompassed in https://github.com/alphagov/smart-answers/pull/5337

Reply:
Agree will try and add it to the flow presenter then – I essential adapted it from the nested partial, clearly not the cleanest way.

Reply:
ah, I just saw your second comment. Would be keen to refactor afterwards then.